Kernel/Arch/aarch64/IRQController.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020, Liav A. <liavalb@hotmail.co.il>
```

Comment:
You should probably put your name in this file :) (unless you don't want to, ofc)

Reply:
I just copied this from the original work and removed some `x86` specific members. So I still think this work belongs to @supercomputer7 ! :)